examples/with-next-js/public/~partytown/debug/partytown-atomics.js

```diff
@@ -0,0 +1,572 @@
+/* Partytown 0.7.4 - MIT builder.io */
```

Comment:
Is the debug folder meant to be included?

Reply:
Yes